### PR TITLE
Bug 1756319: Fix crash with operand resources

### PR DESCRIPTION
**Pull request by spadgett**
* Fixes a crash in some cases displaying the tab
* Fixes an infinite re-render when sorting the table columns

https://bugzilla.redhat.com/show_bug.cgi?id=1756319

/assign @TheRealJon @cyril-ui-developer

### CODE REVIEW

File: frontend/packages/operator-lifecycle-manager/src/components/operand.tsx [hunk 1]
```diff
@@ -466,6 +466,10 @@ export const OperandDetails = connectToModel((props: OperandDetailsProps) => {
   );
 });
 
+const ResourcesTab = (resourceProps) => (
```
Comment (TheRealJon) [inline frontend/packages/operator-lifecycle-manager/src/components/operand.tsx]: nit 
```suggestion
const ResourcesTab = resourceProps => (
```

File: frontend/packages/operator-lifecycle-manager/src/components/k8s-resource.tsx [hunk 2]
```diff
@@ -96,7 +95,7 @@ export const Resources: React.FC<ResourcesProps> = (props) => {
     [kind: string]: { data: K8sResourceKind[] };
   }) => {
     return _.flatMap(resources, (resource, kind: string) =>
-      resource.data.map((item) => ({ ...item, kind })),
+      _.map(resource.data, (item) => ({ ...item, kind })),
```
Comment (TheRealJon) [inline frontend/packages/operator-lifecycle-manager/src/components/k8s-resource.tsx]: nit
```suggestion
      _.map(resource.data, item => ({ ...item, kind })),
```

File: frontend/packages/operator-lifecycle-manager/src/components/operand.tsx (hunk 1 above)
Comment (spadgett) [reply frontend/packages/operator-lifecycle-manager/src/components/operand.tsx]: I think the suggestion will fail prettier

File: frontend/packages/operator-lifecycle-manager/src/components/k8s-resource.tsx (hunk 2 above)
Comment (spadgett) [reply frontend/packages/operator-lifecycle-manager/src/components/k8s-resource.tsx]: Same here, prettier requires the parens

**Pull request merged**